public/app/features/dashboard/dashgrid/PanelChromeAngular.tsx

```diff
@@ -111,19 +111,27 @@ export class PanelChromeAngular extends PureComponent<Props, State> {
   }
 
   componentDidUpdate(prevProps: Props, prevState: State) {
-    if (prevProps.plugin !== this.props.plugin) {
+    const { plugin, height, width, panel } = this.props;
+
+    if (prevProps.plugin !== plugin) {
       this.cleanUpAngularPanel();
       this.loadAngularPanel();
     }
+
+    if (prevProps.width !== width || prevProps.height !== height) {
+      if (this.scopeProps) {
+        this.scopeProps.size.height = height;
+        this.scopeProps.size.width = width;
+        panel.events.emit(PanelEvents.panelSizeChanged);
```

Comment:
should we always do this, or only when angular?

I think better if we make it angular only so react panel plugins do not accidentally get built assuming this event will always exist (even though it will be a long time till we could consider dropping it)

Reply:
This event (and this file where this code is in) is only for angular.

This event is to communicate to the angular directive that it needs to re render

Reply:
ahh right -- it is in `PanelChromeAngular.tsx`  🤦‍♂